faq-v1.md

```diff
@@ -95,3 +97,15 @@ We are eager to discover secure, standards-based ways to execute these processes
 <a name="q11"></a>**Q11: What is verifier impersonation resistance?**
 
 **A11:** Verifier impersonation resistance places requirements on authenticators to reasonably thwart a phishing attack. Phishing attacks attempt to fool an individual into providing valid credentials to an attacker or a rogue site. At AAL3, [SP 800-63B](https://pages.nist.gov/800-63-3/sp800-63b.html) requires authenticators that use a verifier impersonation-resistant authentication protocol to prevent possible phishing attacks. There are a number of ways to do this, including various encryption protocols and digital signature technologies that bind the authenticator output to a specific protected channel. One example of a verifier impersonation-resistant authentication protocol is client-authenticated transport layer security (TLS). In this protocol, the client signs the authenticator output and earlier messages that are unique to the particular TLS connection being negotiated.
+
+<a name="q12"></a>**Q12: Is password expiration no longer recommended?**
+
+**A12:** [SP 800-63B Section 5.1.1.2](https://pages.nist.gov/800-63-3/sp800-63b.html#memsecretver) paragraph 9 states:
+
+>"Verifiers SHOULD NOT require memorized secrets to be changed arbitrarily (e.g., periodically). However, verifiers SHALL force a change if there is evidence of compromise of the authenticator." 
+
+Users tend to choose weaker memorized secrets when they know that they will have to change them in the near future. When those changes do occur, they often select a secret that is similar to their old memorized secret, which provides a false sense of security if any of the previous secrets has been compromised. But if there is evidence that the memorized secret has been compromised, such as by a breach of the verifier's hashed password database or observed fraudulent activity, subscribers should be required to change their memorized secrets. However, this should occur rarely, so that they are less motivated to choose a weak secret with the knowledge that it will only be used for a limited period of time.
+
+<a name="q13"></a>**Q13: Are password composition rules no longer recommended?**
+
+**A13:** [SP 800-63B Section 5.1.1.2](https://pages.nist.gov/800-63-3/sp800-63b.html#memsecretver) paragraph 9 recommends against the use of composition rules (e.g., requiring lower-case, upper-case, digits, and/or special characters) for memorized secrets. These rules provide less benefit than might be expected because users tend to use predictable methods for satisfying these requirements when imposed (e.g., appending a ! to a memorized secret when required to use a special character). The frustration they often face may also cause them to focus on minimally satisfying the requirements rather than devising a memorable but complex secret. Instead, a blacklist of common passwords prevents subscribers from choosing very common values that would be particularly vulnerable, especially to an online attack.
```

Comment:
Would recommend adding something about addressing password re-use specifically, since it's been asked:

> Complexity requirements inadvertently encourage people to use a common password across many different systems, since the burden of complying with complexity requirements is high and creates passwords that are difficult for people to memorize.

Reply:
I'm rewording this one a bit; we want to encourage complexity but not composition rules.

Reply:
I think "memorable but complex" is OK; don't want to revive the discussion on how entropy should be calculated.